api/db/migrate/20230817212021_add_iframe_allows_to_resources.rb

```diff
@@ -0,0 +1,11 @@
+class AddIframeAllowsToResources < ActiveRecord::Migration[6.1]
+  def up
+    add_column :resources, :iframe_allows, :string, array: true, null: false, default: ["fullscreen"]
+    remove_column :resources, :iframe_allow_fullscreen, :boolean
```

Comment:
The client doesn't use this field, so it seemed safe to incorporate into the new one.

Reply:
Always a fan of merging things. Since it isn't being it sounds like we don't need to worry about migrating any false values (since they shouldn't exist).